### ISSUE TITLE
Linux Injector unable to inject since latest update

### ISSUE THREAD

**Issue by ghostlydilemma** [powercord-org/powercord#299]
**Describe the bug**
Running `sudo npm run plug` seems to not work on (at least) Arch Linux (other Linux distros havent been tested yet). 
![image](https://user-images.githubusercontent.com/29519722/75626153-3d19b000-5bc5-11ea-80c4-057420beb614.png)

**To Reproduce**
* Open a console
* Get the newest Powercord version
* Run `npm install`
* Run `sudo npm run plug`
* Alternatively: Run `sudo npm run unplug` - When Powercord is already plugged; won't work either

**Expected behavior**
Inject/Uninject Powercord

**Desktop (please complete the following information):**
 - OS: Linux
 - Discord and Powercord version: Since Powercord can't be injected: Reverting e8848a7671de2eefc996b8b8b43f9e9698069c02 seems to fix the issue
**Additional context**
 - [x] I checked if Discord and/or Powercord are up to date
 - [x] I made sure this is not related to an external plugin or a theme

**Comment by stihlwagon** [comment]
on zorin os it works
at least for me

**Comment by ghostlydilemma** [comment]
```JS
const discordProcess = execSync('ps x')
  .toString()
  .split('\n')
  .map(s => s.split(' ').filter(Boolean))
  .find(p => p[4] && p[4].endsWith('DiscordCanary') && p.includes('--type=renderer'));
```
This section returns undefined on my machine as Discord(Canary) (in any shape or form) cannot be found here (for some reason)

**Comment by cyyynthia** [comment]
Cannot reproduce, using Arch Linux as well.

**Comment by dweam** [comment]
Can reproduce on Gentoo.
![2020-03-01-170345_1292x253_scrot](https://user-images.githubusercontent.com/37590996/75642162-8b9c6e00-5c08-11ea-9be8-b60aa0702f54.png)

**Comment by dweam** [comment]
This seems to happen on systems where the process name is not strictly `DiscordCanary`
i.e: `discord-canary`, `discordcanary`

**Comment by cyyynthia** [comment]
This was fixed in 7bf03d6629c35f0a2525634dd699b537ba4e6946
